### ISSUE TITLE
Improve Multi-language support?

### ISSUE THREAD

**Issue by hellfirehd** [zerratar/SubSync#16]
When specifying the language, does it not follow the normal naming conventions?  e.g. `-lang French,English` should result in `<title> (<year>).fr.srt` or `<title> (<year>).en.srt`.  

I'd like to be able to have it download multiple languages as we often have students from other countries staying with us (mostly Japan) and it's a treat for them to watch a movie with us in English but have the subtitles in their language.  The rest of the time we like subtitles in English (mostly cause volume is low when kids are in bed).  I don't mind running it multiple times, once per language, I just don't want subsequent runs overwriting English with German, etc.

**Comment by zerratar** [comment]
Good idea! And wow, that gotta be exciting having students from different countries. :-)

I myself is a sucker for Japan. My big brother lives in Tokyo, married to a Japanese woman since 7 years back. And I have spent a lot of time studying Japanese. :-) only been there once though.

(Sorry for posting my life story. Hahaha)

**Comment by hellfirehd** [comment]
I hope to get there someday myself.  And Australia and New Zealand and Switzerland... Aw hell... anywhere but here. 😢

**Comment by LechuckThePirate** [comment]
I have the same issue for different reasons... I download normally english/english but my wife doesn't speak english so she likes to see spanish subtitles... having both downloaded would be great.

**Comment by LechuckThePirate** [comment]
I could have a look at that and contribute to your project maybe...
